components/TreasuryAccount/AccountsCompactWrapper.tsx

```diff
@@ -1,27 +1,76 @@
 import AccountsItems from './AccountsItems'
 import HoldTokensTotalPrice from './HoldTokensTotalPrice'
-import useGovernanceAssets from '@hooks/useGovernanceAssets'
 import { ViewState } from './Types'
 import useTreasuryAccountStore from 'stores/useTreasuryAccountStore'
 import useRealm from '@hooks/useRealm'
-import { useEffect } from 'react'
+import React, { useEffect } from 'react'
 import AccountOverview from './AccountOverview'
 import SendTokens from './SendTokens'
 import DepositTokens from './DepositTokens'
+import { PlusIcon } from '@heroicons/react/solid'
+import useQueryContext from '@hooks/useQueryContext'
+import useWalletStore from 'stores/useWalletStore'
+import { useRouter } from 'next/router'
+import Tooltip from '@components/Tooltip'
+const NEW_TREASURY_ROUTE = `/treasury/new`
 
 const AccountsCompactWrapper = () => {
-  const { governedTokenAccounts } = useGovernanceAssets()
+  const router = useRouter()
   const currentView = useTreasuryAccountStore((s) => s.compact.currentView)
   const { resetCompactViewState } = useTreasuryAccountStore()
+  const connected = useWalletStore((s) => s.connected)
   const { symbol } = useRealm()
+  const { fmtUrlWithCluster } = useQueryContext()
+  const { ownCouncilTokenRecord, ownTokenRecord } = useRealm()
+  const goToNewAccountForm = () => {
+    router.push(fmtUrlWithCluster(`/dao/${symbol}${NEW_TREASURY_ROUTE}`))
+  }
+  const isNewAccountRoute = router.route.includes(NEW_TREASURY_ROUTE)
+
+  const canCreateGovernanceUsingCommunityTokens = ownTokenRecord
+
+  const canCreateGovernanceUsingCouncilTokens =
+    ownCouncilTokenRecord &&
+    !ownCouncilTokenRecord.info.governingTokenDepositAmount.isZero()
+
+  const tokenOwnerRecord = canCreateGovernanceUsingCouncilTokens
+    ? ownCouncilTokenRecord
+    : canCreateGovernanceUsingCommunityTokens
+    ? ownTokenRecord
+    : undefined
+  const canCreateGovernance = connected && tokenOwnerRecord
   const getCurrentView = () => {
     switch (currentView) {
       case ViewState.MainView:
         return (
           <>
-            <h3 className="mb-4">Treasury</h3>
+            <h3 className="mb-4 flex items-center">
+              Treasury
+              {!isNewAccountRoute && (
+                <Tooltip
+                  contentClassName="ml-auto"
+                  content={
+                    !canCreateGovernance &&
+                    'Connect your wallet to create new account'
```

Comment:
`Please connect your wallet to create a new treasury account`

Ideally we should split the tooltip messages into two states for all the buttons
1) Wallet not connected - `Please connect your wallet to create a new treasury account`
2) Governance power check - 'You don't have enough governance power to create a new treasury account' 
    The 'governance power' wording should also be generalised using a utility function because for particular use cases we can provide more details like 'not enough governance tokens', 'not enough council tokens', 'not enough governance weight' etc.